feature/interface/otg_tests/interface_ttl_test/egress/README.md

```diff
@@ -0,0 +1,399 @@
+# PF-1.9 Egress handling of TTL
```

Comment:
I think the verification for TTL should be embedded into each of the feature tests (BGP, policy-forwarding, static route with static NHG).  

For example,  the TTL tests for BGP learned routes (without encap) can be moved into the feature/bgp folder (I searched there and the basic bgp tests don't have traffic, so a new test is needed for PF-1.9.(1-3)   
 (I would rename to RT-x.x to fit with the names used for "routing" tests). 

The TTL checking for the policy forwarding and static-route to static NHG scenarios should be moved into the relevant PF and RT tests we already have for each scenario.

Reply:
Dismissing -- I don't think we need TTL handling coverage per routing protocol. TTL handling should be the same for any IPv[46] forwarding, regardless of how it installed in the FIB.

The cases that seem to be material are encapsulation-related.